File: engine/model/src/main/scala/net/server/channel/packet/cash/operation/IncreaseStorageSlotsLarge.scala
package net.server.channel.packet.cash.operation

class IncreaseStorageSlotsLarge(private var _action: Int, private var _cash: Int, private var _mode: Byte, private var _itemId: Int) extends BaseCashOperationPacket(_action = _action) {
  def cash: Int = _cash

  def mode: Byte = _mode

  def itemId: Int = _itemId
}

File: engine/engine-model/src/main/java/net/server/channel/packet/cash/operation/IncreaseStorageSlotsLarge.java
package net.server.channel.packet.cash.operation;

public class IncreaseStorageSlotsLarge extends BaseCashOperationPacket {
   private final Integer cash;

   private final Byte mode;

   private final Integer itemId;

   public IncreaseStorageSlotsLarge(Integer action, Integer cash, Byte mode, Integer itemId) {
      super(action);
      this.cash = cash;
      this.mode = mode;
      this.itemId = itemId;
   }

   public Integer cash() {
      return cash;
   }

   public Byte mode() {
      return mode;
   }

   public Integer itemId() {
      return itemId;
   }
}
